.vscode/launch.json

```diff
@@ -35,7 +35,8 @@
             "mode": "test",
             "program": "${workspaceFolder}/acceptance",
             "args": [
-                "-persist"
+                "-persist",
+                "-tags=@focus"
```

Comment:
Intentional?

Reply:
Yeah, I find myself running single scenario more often than all scenarios, I can revert if this is uncommon...